### PR TITLE
Wmwragg/mention state indicator

### LINKED ISSUE #1760: Chat modal: restyle as per new modal design
Chat modal: restyle as per new modal design

* Design: done assets marked ready

### PR DESCRIPTION
**Pull request by wmwragg**
This closes  #1760.

Signed-off-by: William Wragg wm.wragg@gmail.com

### CODE REVIEW

File: src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css [hunk 1]
```diff
@@ -61,21 +61,43 @@ limitations under the License.
...
+    padding: 0px 4px 0px 4px;
+    z-index: 200;
+}
+
+.mx_RoomTile_highlight .mx_RoomTile_badge:after {
+    content: "";
+    position: absolute;
+    display: block;
+    width: 0;
+	height: 0;
```
Comment (ara4n) [inline src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css]: slightly wonky whitespace here - please no \t's!

**Comment by ara4n** [comment]
hm, am i missing the matrix-react-sdk half of this one?

File: src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css (hunk 1 above)
Comment (wmwragg) [inline src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css]: Sorry about that, not sure how that crept in, I have my editor set to 4 spaces. Fixed

**Comment by wmwragg** [comment]
There is no matrix-react-sdk part for this, it's purely CSS restyling

**Comment by wmwragg** [comment]
No scratch that, I'll dig into why that isn't pushed.

**Comment by wmwragg** [comment]
Yeah sorry, I created both branches, as I thought I'd need to make changes for both sides when I did my initial investigation. But turns out I didn't, but I must have still had it in my head when I made the above comment, sorry. So just to be clear, there is no matrix-react-sdk branch for this, it only required CSS restyle work.

File: src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css [hunk 2]
```diff
@@ -61,21 +61,43 @@ limitations under the License.
...
+}
+
+.mx_RoomTile_highlight .mx_RoomTile_badge:after {
+    content: "";
+    position: absolute;
+    display: block;
+    width: 0;
+    height: 0;
+    margin-left: 7px;
+	border-top: 8px solid #ff0064;
```
Comment (ara4n) [inline src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css]: this still looks to be \t characters - please can we use 4 spaces *everywhere* for tabulation?

**Comment by ara4n** [comment]
otherwise lgtm

File: src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css (hunk 2 above)
Comment (wmwragg) [inline src/skins/vector/css/matrix-react-sdk/views/rooms/RoomTile.css]: This is weird, I'm going to search the code base for tabs, as I'm definitely not putting then in, this code was copied from another section.

**Comment by wmwragg** [comment]
The linked matrix-sdk-branch, is weird. After further investigation, it is requited, and does exist, but says there are no diffs against develop, so I'm not sure whats going on. I'll get back to you.

**Comment by wmwragg** [comment]
OK White space issues fixed. There are a few else where esp. json files. But I'll fix those later. The changes in the matrix-sdk-branch, already seem to be merged into the develop branch. So there is nothing to do there.

**Pull request merged**